workspaces/local-cli/src/shared/spawn-process.ts

```diff
@@ -0,0 +1,24 @@
+import { spawn, SpawnOptions } from 'child_process';
```

Comment:
not sure if there are subtle differences but I think you could have used command-session.ts